packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2720,6 +2722,58 @@ async function handler(
   const metadataFromEvent = videoCallUrl ? { videoCallUrl } : undefined;
 
   try {
+    const hasExistingWorkflow: boolean = eventType.workflows.some((workflow) => {
+      return (
+        workflow.workflow.trigger === WorkflowTriggerEvents.BEFORE_EVENT &&
+        workflow.workflow.time === 1 &&
```

Comment:
```suggestion
        workflow.workflow.time < 12 &&
```

I would prefer sending the reminders only if there is no reminder already sent within 12 hours before the event